package.json

```diff
@@ -46,8 +46,13 @@
     "cssnano": "^4.1.4",
     "dot-prop": ">=5.1.1",
     "eslint": "^7.17.0",
+    "eslint-config-airbnb-base": "^14.2.1",
     "eslint-config-prettier": "^7.1.0",
+    "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.1.3",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-promise": "^4.2.1",
     "gulp": "^3.9.1",
     "gulp-header": "^2.0.5",
     "gulp-image": "^6.2.1",
```

Comment:
Packages that are unused, should be deleted from package.json and package-lock.json.